unreleased_history/performance_improvements/parallel_notify-one_i_LaunchParallelMemTableWriters.md

```diff
@@ -0,0 +1,17 @@
+# Parallelize SetState in LaunchParallelMemTableWriters
```

Comment:
Do you mind giving a shorter summary of the improvement, like improved write throughput to memtable when there's a large number of concurrent writers and allow_concurrent_memtable_write=true? You can add the PR number to the changelog so people can read about the implementation detail there. See more examples in https://github.com/facebook/rocksdb/blob/main/HISTORY.md